README.md

```diff
@@ -48,18 +46,36 @@ If you would like to use an editor like nano, emacs, or vi you have to install i
 
 ### Setting up access credentials
 
-The container comes installed with `libnss-ldap` and `libpam-ldap` if you want to use LDAP for authentication, however this is out of scope of this README. So instead we can do the simple thing of creating new Debian users to the container.
+If you supply environment variables to `docker run`
+
+- AFP_USER        create a user in the container and allow it access to /share
+- AFP_PASSWORD    password
+- AFP_UID         uid of the created user
+- AFP_GID         gid of the created user
+
+Example:
 
 ```bash
-docker exec -ti containername adduser --no-create-home timemachine
+docker run -d \
+-v /mnt/sda1/share:/share \
+--name=afp \
+--net=host \
+-e AFP_USER=$(id -un) \
+-e AFP_PASSWORD=secret \
+-e AFP_UID=$(id -u) \
+-e AFP_GID=$(id -g) \
+cptactionhank/netatalk
 ```
 
-And then the shares could be configured with a share like
+This replaces all occurences of %USER% in afp.conf with AFP_USER
 
 ```ini
-[Secured Share]
-path = /media/secure
-valid users = timemachine
+[Global]
+log file = /var/log/netatalk.log
+
+[share]
+path = /share
```

Comment:
Why change away from `/media` or why not use a directory like `/media/share`. This kinda matches the [FHS 3.0](http://refspecs.linuxfoundation.org/FHS_3.0/fhs-3.0.html#purposeMediaMountPoint) as `/media` contains mount points for removable media which it kinda will be with the described way of mounting an external directory into the container.